libraries/rush-lib/src/schemas/rush-project.schema.json

```diff
@@ -72,6 +72,49 @@
           "disableBuildCacheForOperation": {
             "description": "Disable caching for this operation. The operation will never be restored from cache. This may be useful if this operation affects state outside of its folder.",
             "type": "boolean"
+          },
+
+          "sharding": {
+            "type": "object",
+            "description": "If specified, the operation will be a 'sharded' operation. This means that the operation will be run multiple times in parallel.",
+            "additionalProperties": false,
+            "required": ["count"],
+            "properties": {
+              "count": {
+                "type": "integer",
+                "description": "The number of shards to run. This must be a positive integer."
+              },
+              "shardArgumentFormat": {
+                "type": "string",
+                "description": "A template string that specifies the command-line argument to pass to the operation for each shard. The string may contain the following placeholders: {shardIndex} {shardCount}. Defaults to --shard=\"{shardIndex}/{shardCount}\""
+              },
+              "outputFolderArgument": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "parentFolderNameFormat": {
+                    "type": "string",
+                    "description": "The name of the parent folder that contains the output folders for each shard. The string may contain the following placeholders: {phaseName}. Defaults to \".rush/operation/{phaseName}/shards/{shardIndex}\""
```

Comment:
In practice there is no need to have a placeholder for `phaseName` because this is defined inside of the object where `phaseName` is defined. Therefore it is probably better practice for developers to simply hardcode the name into this string.

Reply:
Yeah, good point. I'll remove this.